sync/command.py

```diff
@@ -440,16 +441,31 @@ def do_landable(git_gecko, git_wpt, *args, **kwargs):
 
     if kwargs["all"] or kwargs["retrigger"]:
         unlandable = unlanded_with_type(git_gecko, git_wpt, wpt_head, prev_wpt_head)
-        status_map = {UnlandableType.ready: "ready",
+
+        status_map = {UnlandableType.ready: "Ready",
```

Comment:
I have a slight preference for seeing status_map and detail_map defined next to the enums they are referring to, rather than in this command.